Source/Core/Core/Boot/Boot.cpp

```diff
@@ -226,7 +226,11 @@ bool CBoot::BootUp()
 	NOTICE_LOG(BOOT, "Booting %s", _StartupPara.m_strFilename.c_str());
 
 	g_symbolDB.Clear();
-	VideoInterface::Preset(_StartupPara.bNTSC);
+
+	// PAL Wii uses NTSC framerate and linecount in 60Hz/480i and 60Hz/480p modes
+	const bool bPAL60 = _StartupPara.bWii && (SConfig::GetInstance().m_SYSCONF->GetData<u8>("IPL.E60") || SConfig::GetInstance().m_SYSCONF->GetData<u8>("IPL.PGS"));
```

Comment:
We can't just disable the combination altogether though because on NTSC consoles, IPL.PGS = 1, IPL.E60 = 0 is the correct combination for 480p, and IIRC some NTSC games even crash when IPL.E60 is enabled. I suppose we can just temporarily set IPL.E60 = 0 when a NTSC game is booted?